tests/bluetooth/iso/src/main.c

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <zephyr/bluetooth/bluetooth.h>
+
+#include "iso_broadcast_src.h"
+#include "iso_broadcast_sink.h"
+#include <zephyr/logging/log.h>
```

Comment:
Newline here

Reply:
This is the format we use for the audio application:
![image](https://user-images.githubusercontent.com/89020256/236787103-797710f0-ef9d-418a-8da1-d25f917b05df.png)